UI/data/locale/en-US.ini

```diff
@@ -480,6 +480,11 @@ ScaleFiltering.Bicubic="Bicubic"
 ScaleFiltering.Lanczos="Lanczos"
 ScaleFiltering.Area="Area"
 
+# blending methods
+BlendingMethod="Blending Method"
+BlendingMethod.Default="Default"
+BlendingMethod.SrgbOff="SRGB Off"
```

Comment:
```suggestion
BlendingMethod.SrgbOff="sRGB Off"
```

For consistency.

Reply:
The capital S is on purpose.

sRGB - The color space.
SRGB - The GPU automatic transfer space conversion. (e.g. D3D: UNORM_SRGB, GL: FRAMEBUFFER_SRGB)